core/selection.js

```diff
@@ -943,10 +943,24 @@
 				// https://dev.ckeditor.com/ticket/14407 - Don't even let anything happen if the selection is in a non-editable element.
 				editable.attachListener( editable, 'keydown', function( evt ) {
```

Comment:
Extracting the whole function into named private function (like it has been done for above listeners) should improve readability.